rfcs/0008-leaderless-dns.md

```diff
@@ -0,0 +1,238 @@
+# Distributed DNS Load Balancing
+
+- Feature Name: Distributed DNS Load Balancing
+- Start Date: 2024-01-17
+- RFC PR: [Kuadrant/architecture#0008](https://github.com/Kuadrant/architecture/pull/55)
+- Issue tracking: [Kuadrant/architecture#0000](https://github.com/Kuadrant/architecture/issues/0000)
```

Comment:
Might be good to add an issue to the arch repo that is for `Distributed DNS` and link to the two epics we created yesterday